doc/sphinx/reference/vcl_var.rst

```diff
@@ -536,8 +539,11 @@ req.ttl
 
 	Writable from: client
 
+	Unsettable from: client
 
 	Upper limit on the object age for cache lookups to return hit.
+	Lookups always miss when req.ttl is set to 0s. Negative values
```

Comment:
Maybe it is worth noting that setting `req.ttl` to `1ms` will keep request coalescing intact, making this a better way of making sure that a fresh object is in the cache, without the risk of inserting many. `set req.ttl = 1ms;` is also better than setting `hash_always_miss` in many (all?) cases. Of course, such notes can be made elsewhere, in a different PR. Still, are we sure that _n_ requests coming in at the same time (before the first one finishes `sub vcl_backend_response` with the subsequent _unbusy_), all getting a `set req.ttl = 0s`, will produce _n_ backend requests, potentially inserting _n_ objects?